### DISCUSSION TITLE
Are custom members available?

### DISCUSSION THREAD

**Discussion by Harold7665** [atoti/atoti#669] [Q&A] [answered]
Hello, is this ActivePivot feature available on atoti: https://activeviam.com/activepivot/latest/docs/cube/member_properties/ ?

**Comment by Skyscimitar** [comment, marked as answer]
Hello,

This feature is not available in the Python API at the moment. Could you explain your use-case for them please? If it's an internal use-case you can add it on this issue: https://github.com/activeviam/atoti/issues/3850.

**Reply by Harold7665** [reply]
Hey, this is based on a prospect feedback, I keep you in touch.
